content/recipes/swc.md

```diff
@@ -95,3 +95,90 @@ If your ORM does not provide a similar workaround, you can define the wrapper ty
  */
 export type WrapperType<T> = T; // WrapperType === Relation
 ```
+### Vitest
+
+[Vitest](https://vitest.dev/) is a fast and lightweight test runner designed to work with Vite. It provides a modern, fast, and easy-to-use testing solution that can be integrated with NestJS projects.
+
+#### Installation
+
+To get started, first install the required packages:
+
+```bash
+$ npm i --save-dev vitest unplugin-swc @swc/core @vitest/coverage-c8
+```
+
+#### Configuration
+
+Create a vitest.config.ts file in the root directory of your application with the following contents:
+
+```ts
+import swc from 'unplugin-swc';
+import { defineConfig } from 'vite';
+
+export default defineConfig({
+  test: {
+    globals: true,
+    alias: {
+      '@src': './src',
+      '@test': './test',
```

Comment:
We might need to add a blockquote (hint) below this snippet just to explain that "alias"/"resolve" are optional and serve as an example of how to use ts paths. To make things even simpler, we could split this example into 2 - a simple one without aliases and the other one with them